realm/realm-library/src/androidTest/java/io/realm/DynamicRealmObjectTests.java

```diff
@@ -1220,7 +1220,7 @@ public void toString_test() {
         // Checks that toString() doesn't crash, and does simple formatting checks. We cannot compare to a set String as
         // eg. the byte array will be allocated each time it is accessed.
         String str = dObjTyped.toString();
-        assertTrue(str.startsWith("AllJavaTypes = ["));
+        assertTrue(str.startsWith("AllJavaTypes = dyn["));
```

Comment:
Also nitpicking I would personally prefer `AllJavaTypes (Proxy) = [ ]` and `AllJavaTypes (DynamicObject) = [ ]` since the right-hand side of the `=` is the objects data and the left hand side is the meta information.